Bixal/move.mil
# Parse all xml

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Refactor "truncate" command, to delete just one location type.
- Creates another custom module for managing the locations data.
- Creates a form to upload the locations file (xml) with the latest changes.
- Updates, creates, and deletes locations in batch processes.

## Testing

To verify the changes proposed in this pull request…

1. Import configuration and clear cach
2. go to http://move.mil.localhost:8000/admin/config/locations/manager
3. Upload a xml file and click on Update locations
4. Wait like 15 minutes, and check taht the expected locations were updated, created, or deleted.

## Screenshots
Updating progess
![updatingprogress](https://user-images.githubusercontent.com/37001835/52582283-dc250b80-2dfa-11e9-959c-773474962d54.png)

Finish With Warnings
![finishwithwarnings](https://user-images.githubusercontent.com/37001835/52582312-efd07200-2dfa-11e9-9e99-559684a504fe.png)
